Core/SaveState.cpp

```diff
@@ -621,7 +604,14 @@ namespace SaveState
 
 			case SAVESTATE_SAVE:
 				INFO_LOG(SAVESTATE, "Saving state to %s", op.filename.c_str());
-				result = CChunkFileReader::Save(op.filename, g_paramSFO.GetValueString("TITLE"), PPSSPP_GIT_VERSION, state);
+				title = g_paramSFO.GetValueString("TITLE");
+				if (title.empty()) {
+					// Homebrew title
+					title = PSP_CoreParameter().fileToStart;
+					std::size_t lslash = title.find_last_of("/");
```

Comment:
Hm, I guess the path is always normalized here so we don't have to care about '\\'..